src/server-extension/query/event.ts

```diff
@@ -3,4 +3,15 @@ export const buyEvent = `SELECT
     COALESCE(MAX(e.meta::bigint), 0) as max
     FROM event e
     LEFT JOIN nft_entity ne on ne.id = e.nft_id
-    WHERE e.interaction = 'BUY' AND ne.collection_id = $1;`
\ No newline at end of file
+    WHERE e.interaction = 'BUY' AND ne.collection_id = $1;`
+
+export const collectionEventHistory = `SELECT
+	ce.id as collection_id,
+	DATE(e.timestamp),
+	count(e) as freq
+FROM nft_entity ne
+JOIN collection_entity ce on ce.id = ne.collection_id
+JOIN event e on e.nft_id = ne.id
+WHERE e.interaction = 'BUY' and ce.id = $1 and e.timestamp >= NOW() - INTERVAL '365 DAY'
+GROUP BY ce.id, DATE(e.timestamp)
+ORDER BY ce.id`
```

Comment:
👀 you select single id and then you use order for it? It does not make sense 

```suggestion
ORDER BY DATE(e.timestamp)`
```